generators/client/templates/src/main/webapp/app/components/alert/_alert-error.directive.js

```diff
@@ -0,0 +1,109 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .component('jhAlertError', jhAlertError);
+
+    function jhAlertError () {
+        var component = {
+            template: '<div class="alerts" ng-cloak="">' +
+                            '<div ng-repeat="alert in alerts" ng-class="[alert.position, {\'toast\': alert.toast}]">' +
```

Comment:
Yes it's `vm.alerts`, sorry

I"ll add `[ci skip]` next time for those type of commits, didn't know about that trick !